src/main/java/io/swagger/swaggerhub/plugin/requests/SaveSCMPluginConfigRequest.java

```diff
@@ -0,0 +1,224 @@
+package io.swagger.swaggerhub.plugin.requests;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.swaggerhub.plugin.requests.dtos.SCMIntegrationPluginConfiguration;
+
+/**
+ * Configuration class used to store details required to create a request to SwaggerHub to create an SCM integration plugin.
+ * The inner builder class follows the builder pattern which aids building the SaveSCMPluginConfigRequest object.
+ * {@link SCMIntegrationPluginConfiguration} represents the object when being sent as part of a request body.
+ */
+public class SaveSCMPluginConfigRequest {
+
+    /*
+    Path Params
+     */
+    private String apiOwner;
```

Comment:
The path params here are largely the same as the path params in `SwagggerHubRequest`. Could do some inheritance here (maybe create an abstract `Request` class?)  to avoid repetition?